lhotse/workflows/activity_detection/README.md

```diff
@@ -0,0 +1,162 @@
+# Activity Detection
+
+The Activity Detection module provides tools for detecting activity in audio recordings. It includes the following components:
+
+- `ActivityDetector` base class, designed to facilitate the integration of new activity detection methods in the future.
+- A runner class named `ActivityDetectionProcessor` for parallel execution of activity detection on the `RecordingSet``.
+- Two classes, `SileroVAD8k` and `SileroVAD16k`, which integrate the [Silero VAD](https://github.com/snakers4/silero-vad) model for activity detection.
+
+
+## Example usage in command line interface (CLI)
+
+1. Read the Help
+
+    ```bash
+    lhotse workflows activity-detection --help
+    ```
+
+2. Please note: if you are using the activity-detection script for the first time with any detection model, be sure to add the `--force_download` flag to preload the model state on your local machine from the appropriate source. Like this:
+
+    ```bash
+    lhotse workflows activity-detection \
+    --model-name silero_vad_16k \
+    ...\
+    --force_download
+    ```
```

Comment:
This is not really what I meant about the `force_download`. I meant that the user would just run the command once, with or without this optional flag.

**Case 1:** If the flag is not set, then use existing downloaded model, or download if it does not exist.

**Case 2:** If the flag is set, clear cache and re-download model before starting the VAD processing.

Reply:
Yes, I really misinterpreted that. I've made the appropriate change.